src/read_cache_fs_extension.cpp

```diff
@@ -67,6 +67,12 @@ static void LoadInternal(DatabaseInstance &instance) {
     }
   }
 
+  auto &config = DBConfig::GetConfig(instance);
+  config.AddExtensionOption(
+      "cached_http_cache",
+      "The diskcache directory temporarily stores cached data ",
```

Comment:
`diskcache` -> `disk cache`, follow the same naming style elsewhere

Reply:
`OnDiskCacheConfig` should have the same default value as config, please use a const variable at both places.